pkg/security/secl/rules/ruleset.go

```diff
@@ -674,6 +678,8 @@ func (rs *RuleSet) Evaluate(event eval.Event) bool {
 		}
 	}
 
+	rs.eventCollector.CollectEvent(rs, event, result)
```

Comment:
I would like a comment here saying that this func is a no-op in non-functional test builds because it can be hard to follow build tags (unlike interfaces)

Reply:
added in last commit